chromadb/segment/impl/metadata/sqlite.py

```diff
@@ -235,6 +271,19 @@ def _update_metadata(self, cur: Cursor, id: int, metadata: UpdateMetadata) -> No
             sql, params = get_sql(q)
             cur.execute(sql, params)
 
+        lists_to_update = [k for k, v in metadata.items() if isinstance(v, list)]
```

Comment:
is a metadata key for a list is updated, first delete the old list before inserting the new elements